missionbit/settings.py

```diff
@@ -62,21 +64,27 @@
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
+                'social_django.context_processors.backends',  # <- Social Django Oauth Google
+                'social_django.context_processors.login_redirect', # <- Social Django Oauth Google
             ],
         },
     },
 ]
 
-WSGI_APPLICATION = 'missionbit_portal.wsgi.application'
+WSGI_APPLICATION = 'missionbit.wsgi.application'
 
 
 # Database
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2', # <- Postgeres Setup
+        'NAME' : '', # <- Postgeres Setup (source into bash later)
+        'USER' : '', # <- Postgeres Setup (source into bash later)
+        'PASSWORD': '', # <- Postgeres Setup (source into bash later)
+        'HOST': 'localhost', # <- Postgeres Setup 
+        'PORT': '5432' # <- Postgeres Setup 
```

Comment:
Rather than sourcing into bash it's pretty common to pick up configuration from the environment and for local dev it's convenient to use a `.env` file (which should be in `.gitignore` since these files often have secrets in them). I think https://github.com/theskumar/python-dotenv is the package that most folks use to manage that with Python

Reply:
Got it